composer.json

```diff
@@ -4,32 +4,34 @@
     "description": "Temporary storage adapter factory for fluent migration to laminas-cache v3 when working with laminas components which depend on laminas-cache",
     "license": "BSD-3-Clause",
     "require": {
-        "php": "^7.3 || ~8.0.0 || ~8.1.0",
+        "php": "~8.0.0 || ~8.1.0 || ^8.2.0",
```

Comment:
this will allow PHP 8.3 as well, please modify the constraint to be in-sync with the two others in the same line

Reply:
done